docs/source/sft_trainer.md

```diff
@@ -54,6 +54,13 @@ The above snippets will use the default training arguments from the [`SFTConfig`
 
 ## Advanced usage
 
+### Train on assistant messages only
+
+To train on assistant messages only, use a [conversational](dataset_formats#conversational) [language modeling](dataset_formats#language_modeling) dataset and set `assistant_only_loss=True` in the [`SFTConfig`]. This setting ensures that loss is computed **only** on the assistant responses, ignoring user and system and user messages.
+
+> [!WARNING]
+> This functionality is only available for chat templates that support returning the  assistant tokens mask via the `{% generation %}` keyword.
```

Comment:
Maybe point to an example chat template that has this feature?

Reply:
Ah yes good idea